File: backend/app/models/line_binding_user.py
from models.database import db
from datetime import datetime

class LineBindingUser(db.Model):
    __tablename__ = "line_binding_user"

    line_id = db.Column(db.String(120), primary_key=True, unique=True)
    created_at = db.Column(db.DateTime, default=datetime.utcnow)
    user_id = db.Column(db.Integer, db.ForeignKey('users.id'), nullable=False)

    user = db.relationship('User', back_populates='line_binding_user')

File: backend/golang-app/models/line_binding_user.go
package models

import (
	"time"
)

type LineBindingUser struct {
	ID        uint      `gorm:"primaryKey"`
	LineID    string    `gorm:"size:120;unique"`
	CreatedAt time.Time `gorm:"autoCreateTime"`
	UserID    uint      `gorm:"not null"`
	User      User      `gorm:"foreignKey:UserID"`
}

func (LineBindingUser) TableName() string {
	return "line_binding_user"
}
